### PR TITLE
Expand-the-VMware-Integration

### PR DESCRIPTION
**Pull request by mihaitusa**
Adding guest content.
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
@Bonczidai  Please review it!

File: content/io/cloudslang/virtualization/vmware/virtual_machines/create_virtual_machine.sl [hunk 1]
```diff
@@ -87,22 +95,33 @@ operation:
         overridable: false
     - data_center_name
     - dataCenterName:
-        default: ${get("data_center_name", None)}
+        default: ${data_center_name}
```
Comment (Bonczidai) [inline content/io/cloudslang/virtualization/vmware/virtual_machines/create_virtual_machine.sl]: Y? won't it fails with a nasty Python error if `data_center_name` is not supplied?

File: test/io/cloudslang/virtualization/vmware/guest/test_customize_linux_guest.sl [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            - trust_everyone
+            - virtual_machine_name
+            - computer_name
+            - domain
+            - ip_address
+            - subnet_mask
+            - default_gateway
+            - hw_clock_utc
+            - time_zone
+
```
Comment (Bonczidai) [inline test/io/cloudslang/virtualization/vmware/guest/test_customize_linux_guest.sl]: maybe remove newline

File: test/io/cloudslang/virtualization/vmware/guest/test_customize_linux_guest.sl [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        publish:
+          - return_result
+          - return_code
+          - exception : ${exception if exception != None else ''}
```
Comment (Bonczidai) [inline test/io/cloudslang/virtualization/vmware/guest/test_customize_linux_guest.sl]: `get()`

File: test/io/cloudslang/virtualization/vmware/guest/test_customize_linux_guest.sl [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+          FAILURE: CUSTOMIZE_LINUX_GUEST_FAILURE
+
+    - check_result:
+        do:
+          lists.compare_lists:
+            - list_1: ${[str(exception), int(return_code)]}
+            - list_2: ['', 0]
+        navigate:
+          SUCCESS: get_text_occurrence
+          FAILURE: CHECK_RESPONSES_FAILURE
```
Comment (Bonczidai) [inline test/io/cloudslang/virtualization/vmware/guest/test_customize_linux_guest.sl]: maybe `CHECK_RESULT_FAILURE` to be aligned with task name

File: content/io/cloudslang/virtualization/vmware/virtual_machines/create_virtual_machine.sl (hunk 1 above)
Comment (mihaitusa) [inline content/io/cloudslang/virtualization/vmware/virtual_machines/create_virtual_machine.sl]: Will fail in CLI... the required input value not provided....

**Comment by Bonczidai** [comment]
waiting for new builder (with vmware actions)

File: test/io/cloudslang/virtualization/vmware/guest/test_customize_linux_guest.sl (hunk 3 above)
Comment (mihaitusa) [inline test/io/cloudslang/virtualization/vmware/guest/test_customize_linux_guest.sl]: yes... that will be good!

**Comment by mbettan** [comment]
Can we commit it? I need them for my use cases

**Comment by mihaitusa** [comment]
@Bonczidai Added the requested test flow (test_virtual_machine_lifecycle)

**Comment by mihaitusa** [comment]
Waits for https://github.com/CloudSlang/score-actions/pull/104 to be merged && released to MVNC

**Comment by mihaitusa** [comment]
@Bonczidai  Now we have score-vmware 0.0.4 on MC. It can be merged. WDYT?

**Pull request merged**